### PR TITLE
016/master

### PR DESCRIPTION
**Pull request by BenGitsCode**
Refresh for cohort 016

-still debugging solution branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ This diagnostic measures your ability to create and use Ember Objects and
 1.  Install dependencies by running `npm install` and `bower install`.
 1.  Create and checkout a new branch, named `response`.
 1.  Follow the directions given in [diagnostic.js](./public/diagnostic.js).
+1.  All diagnostic responses are to be made in [diagnostic.js](./public/diagnostic.js).
+1.  Do not work inside the actual ember app in this repository.
```
Comment (raq929) [inline README.md:20]: I think this is overkill. :)

**Comment by raq929** [comment]
What do you mean by phantomjs blows up?
Travis is saying that tests are failing, which is a separate issue.

**Comment by BenGitsCode** [comment]
@raq929 RE:[phantomjs](https://github.com/ga-wdi-boston/ember-object-diagnostic/pull/100#issuecomment-288243149) that was my misunderstanding; phantomjs (i think) was blowing up because I missing a test file, and I think the tests are failing because that file is a test file. This repo doesn't have tests for the diagnostic (just the defaults generated by the ember app) so it won't effect the delivery in this iteration. I've opened an issue with what I believe to be the missing test files.

File: README.md (hunk 1 above)
Comment (BenGitsCode) [reply README.md:20]: I'm inclined to agree but Jeff said to be more explicit. Maybe it would have combined the two into something more concise :)

**Pull request merged**